it/controllers/benefits/income/IncomeTaxBenefitsAmountControllerISpec.scala

```diff
@@ -0,0 +1,546 @@
+    val expectedErrorIncorrectFormat = "Enter the amount of Income Tax paid by your employer in the correct format"
+    val expectedErrorOverMaximum = "The Income Tax paid by your employer must be less than £100,000,000,000"
+  }
+
+  object ExpectedIndividualCY extends SpecificExpectedResults {
+    val expectedTitle = "How much of your Income Tax did your employer pay?"
+    val expectedHeading = "How much of your Income Tax did your employer pay?"
+    val expectedErrorTitle = s"Error: $expectedTitle"
+    val expectedErrorNoEntry = "Enter the amount of Income Tax paid by your employer"
+    val expectedErrorIncorrectFormat = "Enter the amount of Income Tax paid by your employer in the correct format"
+    val expectedErrorOverMaximum = "The Income Tax paid by your employer must be less than £100,000,000,000"
+  }
+
+  object ExpectedAgentEN extends SpecificExpectedResults {
+    val expectedTitle = "How much of your client’s Income Tax did their employer pay?"
+    val expectedHeading = "How much of your client’s Income Tax did their employer pay?"
+    val expectedErrorTitle = s"Error: $expectedTitle"
+    val expectedErrorNoEntry = "Enter the amount of Income Tax paid by your client’s employer"
+    val expectedErrorIncorrectFormat = "Enter the amount of Income Tax paid by your client’s employer in the correct format"
+    val expectedErrorOverMaximum = "The Income Tax paid by your client’s employer must be less than £100,000,000,000"
+  }
+
+  object ExpectedAgentCY extends SpecificExpectedResults {
+    val expectedTitle = "How much of your client’s Income Tax did their employer pay?"
+    val expectedHeading = "How much of your client’s Income Tax did their employer pay?"
+    val expectedErrorTitle = s"Error: $expectedTitle"
+    val expectedErrorNoEntry = "Enter the amount of Income Tax paid by your client’s employer"
+    val expectedErrorIncorrectFormat = "Enter the amount of Income Tax paid by your client’s employer in the correct format"
+    val expectedErrorOverMaximum = "The Income Tax paid by your client’s employer must be less than £100,000,000,000"
+  }
+
+  val userScenarios: Seq[UserScenario[CommonExpectedResults, SpecificExpectedResults]] = {
+    Seq(UserScenario(isWelsh = false, isAgent = false, CommonExpectedEN, Some(ExpectedIndividualEN)),
+      UserScenario(isWelsh = false, isAgent = true, CommonExpectedEN, Some(ExpectedAgentEN)),
+      UserScenario(isWelsh = true, isAgent = false, CommonExpectedCY, Some(ExpectedIndividualCY)),
+      UserScenario(isWelsh = true, isAgent = true, CommonExpectedCY, Some(ExpectedAgentCY)))
+  }
+
+  ".show" should {
+
+    userScenarios.foreach { user =>
+      import Selectors._
+      import user.commonExpectedResults._
+
+      s"language is ${welshTest(user.isWelsh)} and request is from an ${agentTest(user.isAgent)}" should {
+        "render the how much of your client’s Income Tax did their employer pay?' page with an empty amount field" which {
```

Comment:
for this scenario can you add a elementNotOnPageCheck to make sure that the "If it was not XXX" line is not on the page